ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -96,6 +97,10 @@ public static final class Headers {
      * {@code "Date"}
      */
     public static final String DATE = "Date";
+    /**
+     * {@code "Expect"}
+     */
+    public static final String EXPECT = "Expect";
```

Comment:
Please add some comments here that this is only supported for named blob Put

Reply:
Sure, updated.